docs/framework/migration-guide/migrating-from-the-net-framework-1-1.md

```diff
@@ -1,61 +1,65 @@
-  
- To assess the impact of types and members that have been deprecated since the release of the [!INCLUDE[net_v20SP1_short](../../../includes/net-v20sp1-short-md.md)], see [What's Obsolete in the Class Library](../../../docs/framework/whats-new/whats-obsolete.md). Review the lists of obsolete types and member for the [!INCLUDE[net_v20SP1_short](../../../includes/net-v20sp1-short-md.md)], the [!INCLUDE[net_v35_short](../../../includes/net-v35-short-md.md)], and the [!INCLUDE[net_v40_short](../../../includes/net-v40-short-md.md)].
+
+[!INCLUDE[win7](../../../includes/win7-md.md)] and later versions of the Windows operating system do not support the [!INCLUDE[net_v11_long](../../../includes/net-v11-long-md.md)]. As a result, applications that target the [!INCLUDE[net_v11_short](../../../includes/net-v11-short-md.md)] will not run without modification on [!INCLUDE[win7](../../../includes/win7-md.md)] or later operating system versions. This topic discusses the steps required to run an application that targets the [!INCLUDE[net_v11_short](../../../includes/net-v11-short-md.md)] under [!INCLUDE[win7](../../../includes/win7-md.md)] and later versions of the Windows operating system. For more information about the [!INCLUDE[net_v11_long](../../../includes/net-v11-long-md.md)] and [!INCLUDE[win8](../../../includes/win8-md.md)], see [Running .NET Framework 1.1 Apps on Windows 8 and later versions](../../../docs/framework/install/run-net-framework-1-1-apps.md).
+
+## Retargeting or Recompiling
+
+There are two ways to get an application that was compiled using the [!INCLUDE[net_v11_short](../../../includes/net-v11-short-md.md)] to run on [!INCLUDE[win7](../../../includes/win7-md.md)] or a later Windows operating system:
+
+-   You can retarget the application to run under [!INCLUDE[net_v40_long](../../../includes/net-v40-long-md.md)]. Retargeting requires that you add a [\<supportedRuntime>](../../../docs/framework/configure-apps/file-schema/startup/supportedruntime-element.md) element to the application's configuration file that allows it to run under [!INCLUDE[net_v40_short](../../../includes/net-v40-short-md.md)]. Such a configuration file takes the following form:
```

Comment:
unrelated to your PR: the two tokens here can be replaced with ".NET Framework 4 and later versions"